src/domain/community/community/HomePageContributors/useHomePageContributors.ts

```diff
@@ -1,81 +1,71 @@
 import { useMemo } from 'react';
 import { shuffle } from 'lodash';
 import { useUserContext } from '../../../../hooks';
-import { useContributingUsersQuery, useOrganizationsListQuery } from '../../../../hooks/generated/graphql';
-import useServerMetadata from '../../../../hooks/useServerMetadata';
+import { useContributingUsersQuery, useContributingOrganizationsQuery } from '../../../../hooks/generated/graphql';
 import { COUNTRIES_BY_CODE } from '../../../../models/constants';
-import getMetricCount from '../../../platform/metrics/utils/getMetricCount';
 import { buildOrganizationUrl, buildUserProfileUrl } from '../../../../common/utils/urlBuilders';
 import { getVisualAvatar } from '../../../../common/utils/visuals.utils';
 import { ContributorCardProps } from '../../../../common/components/composite/common/cards/ContributorCard/ContributorCard';
 import { WithId } from '../../../../types/WithId';
 import { AuthorizationCredential } from '../../../../models/graphql-schema';
 
-const MAX_USERS_TO_SHOW = 12;
-const MAX_ORGANIZATIONS_TO_SHOW = 12;
+const MAX_ITEMS_TO_SHOW = 20;
+const HALF_MAX_ITEMS_TO_SHOW = MAX_ITEMS_TO_SHOW / 2; // If logged in, show half users half organizations
 
 const useHomePageContributors = () => {
-  const { user, isAuthenticated } = useUserContext();
+  const { isAuthenticated } = useUserContext();
 
-  const { data: usersData, loading } = useContributingUsersQuery({
-    fetchPolicy: 'cache-and-network',
-    skip: !isAuthenticated,
-  });
+  const organizationsCountLimit = isAuthenticated ? HALF_MAX_ITEMS_TO_SHOW : MAX_ITEMS_TO_SHOW;
 
-  const { data: organizationsData, loading: loadingOrganizations } = useOrganizationsListQuery({
-    fetchPolicy: 'cache-and-network',
+  const { data: organizationsData, loading: loadingOrganizations } = useContributingOrganizationsQuery({
     variables: {
-      limit: MAX_ORGANIZATIONS_TO_SHOW,
+      limit: organizationsCountLimit,
       shuffle: true,
       filterCredentials: [AuthorizationCredential.HubHost, AuthorizationCredential.ChallengeLead],
     },
```

Comment:
skip the organization query if the user context is still loading - `isAuthenticated` is not initialized correctly at this time and full amount of orgs are going to be queried instead of half.